testing/mod.ts

```diff
@@ -170,21 +252,33 @@ async function runTestsParallel(
 async function runTestsSerial(
   stats: TestStats,
   tests: TestDefinition[],
-  exitOnFail: boolean
+  exitOnFail: boolean,
+  disableLog: boolean
 ): Promise<void> {
   for (const { fn, name } of tests) {
-    // See https://github.com/denoland/deno/pull/1452
-    // about this usage of groupCollapsed
-    console.groupCollapsed(`test ${name} `);
+    // Displaying the currently running test if silent mode
+    if (disableLog) {
+      print(`${yellow("RUNNING")} ${name}`, false);
+    }
     try {
+      let start, end;
+      start = performance.now();
       await fn();
+      end = performance.now();
+      if (disableLog) {
+        // Rewriting the current prompt line to erase `running ....`
+        print("\x1b[2K\r", false);
+      }
       stats.passed++;
-      console.log("...", GREEN_OK);
-      console.groupEnd();
+      print(
+        GREEN_OK + "     " + name + " " + promptTestTime(end - start, true)
+      );
     } catch (err) {
-      console.log("...", RED_FAILED);
-      console.groupEnd();
-      console.error(err.stack);
+      if (disableLog) {
+        print("\x1b[2K\r", false);
```

Comment:
ANSI escape sequences are hard to read. Ideally you'd define them in a constant. Something like:
```js
const CLEAR_LINE = "\x1b[2K";
``` 
(I don't actually know what that ANSI sequence means, I'm just giving an example)

Reply:
This is used to erase the the current line: http://ascii-table.com/ansi-escape-sequences-vt-100.php